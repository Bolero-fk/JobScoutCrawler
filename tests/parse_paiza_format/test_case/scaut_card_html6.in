<div class="MessageCardHeader"><img class="ScoutImage" src="sample6.jpg" alt="スカウト一覧画像"></div><div class="MessageCardBody"><div class="MessageCardTitle"><div class="MessageCardTitleName">株式会社あいうえお</div><div class="MessageUnReadPoint"></div></div><div class="MessageCardText">サンプル説明文6サンプル説明文6サンプル説明文6サンプル説明文6サンプル説明文6サンプル説明文6</div><ul class="MessageCardStatus"><li class="MessageCardStatusItem"><i class="fas fa-yen-sign"></i>  経験と能力に応じて決定</li><li class="MessageCardStatusItem"><i class="fas fa-map-marker-alt"></i>  福岡県</li><li class="MessageCardStatusItemSub"><i class="fas fa-code"></i> - - -</li></ul><div class="MessageCardDate"><span style="margin-right: 0.25rem;">2023/03/09</span><span style="color: rgb(255, 147, 0);">残り6日</span></div></div>