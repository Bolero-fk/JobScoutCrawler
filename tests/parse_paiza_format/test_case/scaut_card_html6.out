{"company_name": "株式会社あいうえお", "description": "サンプル説明文6サンプル説明文6サンプル説明文6サンプル説明文6サンプル説明文6サンプル説明文6", "min_salary": null, "max_salary": null, "location": "福岡県", "using_lang": "- - -", "recieve_date": "2023/03/09", "remaining_days": 6}